Move External Identities APIs to v1.0 (#12210)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/authenticationflowspolicy.md

---
title: "authenticationFlowsPolicy resource type"
description: "Represents the policy configuration of self-service sign-up experience at a tenant level that lets external users request to sign up for approval. "
author: "linkhp"
localization_priority: Priority
ms.prod: "identity-and-sign-in"
doc_type: resourcePageType
---

# authenticationFlowsPolicy resource type

Namespace: microsoft.graph

> Represents the [policy configuration of self-service sign-up experience](../resources/selfservicesignupauthenticationflowconfiguration.md) at a tenant level that lets external users request to sign up for approval. It contains information about the ID, display name, and description, and indicates whether self-service sign up is enabled for the policy.

Reviewer: ```suggestion
Represents the [policy configuration of self-service sign-up experience](../resources/selfservicesignupauthenticationflowconfiguration.md) at a tenant level that lets external users request to sign up for approval. It contains information, such as the identifier, display name, and description, and indicates whether self-service sign-up is enabled for the policy.
```

## Methods

| Method       | Return Type  |Description|
|:---------------|:--------|:----------|
|[Get authentication flows policy](../api/authenticationflowspolicy-get.md)|authenticationFlowsPolicy|Get the authentication flows policy configuration.|
|[Update authentication flows policy](../api/authenticationflowspolicy-update.md)|authenticationFlowsPolicy|Update the authentication flows policy configuration.|

## Properties

|Property|Type|Description|
|:-------|:---|:----------|

> |id|String| Inherited property. The ID of the authentication flows policy. Optional. Read-only.

Reviewer: ```suggestion
|id|String| Inherited property. The identifier of the authentication flows policy. Optional. Read-only.
```

> |displayName|String| Inherited property. The human-readable name of the policy. This property is not a key. Optional. Read-only.|

Reviewer: What is the purpose of describing that the property is not a key?
